### ISSUE TITLE
Testing mlmodel prediction

### ISSUE THREAD

**Issue by evanb2** [tf-coreml/tf-coreml#187]
I tried using the example code from `inception_v3.ipynb` but I'm executing my conversion script from within a Docker container and so `mlmodel.predit()` throws the following error:
```
Exception: Model prediction is only supported on macOS version 10.13 or later.
```
How can I get around this? I just want a simple way to run a prediction through the mlmodel to test it's accuracy. My python knowledge is very limited.

Thanks!

**Comment by aseemw** [comment]
Coremltools calls out to the CoreML framework (through pybind c++ bindings) for actually running the engine and making predictions. 

CoreML framework is only available on MacOS 10.13 or later. 
I am not familiar with Docker, but can you make sure that you have access to the CoreML framework on it? (not sure how one would do that though)

**Comment by evanb2** [comment]
@aseemw Ok so I ended up removing Docker as it was unnecessary to the project. I'm no longer having that environment issue.

However, when I try to run a test prediction I get this error:
```
Traceback (most recent call last):
  File "convert_to_coreml.py", line 41, in <module>
    probs = coreml_output['final_result__0'].flatten()
AttributeError: 'dict' object has no attribute 'flatten'
```
